documentation/sphinx/source/administration.rst

```diff
@@ -799,3 +799,16 @@ Upgrading from Older Versions
 -----------------------------
 
 Upgrades from versions older than 5.0.0 are no longer supported.
+
+Version-specific notes on downgrading
+===================================
+
+In general, downgrades between patch releases (i.e. 6.2.x - 6.1.x) are not supported.
+
+Downgrading from 6.3.13 - 6.2.33
+--------------------------------
+After upgrading from 6.2 to 6.3, the option of rolling back and downgrading to return to 6.2 is still possible, given that the following conditions are met:
+
+* The 6.3 cluster cannot have ``TLogVersion`` greater than V4 (6.2).
+* The 6.3 cluster cannot use storage engine types that are not ``ssd-1``, ``ssd-2``, or ``memory``.
+* The 6.3 cluster must not have any key servers serialized with tag encoding. The ``TAG_ENCODE_KEY_SERVERS`` fdbclient knob must not be set to true at any point in time.
```

Comment:
Is the second sentence a separate condition for the first, or is it just providing a definition for the first?

Reply:
The conditions are all separate

Reply:
I'm not sure, then, that the first sentence is going to be very clear to users. What does it mean for the key servers to be serialized with the tag encoding?

Reply:
Sorry, I think i misinterpreted the comment. You are correct that the second sentence is just providing a definition for the first. I originally thought you meant the point above involving storage engines